app/(admin)/admin/notifications/page.tsx

```diff
@@ -1,16 +1,52 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import { JSX } from 'react';
+'use client';
+import { JSX, useState } from 'react';
+import { sendEmailNotifications } from './actions/sendEmailNotification';
+import React from 'react';
 
 /**
  * The admin home page.
  * @returns The rendered AdminHome page.
  */
 const AdminNotifications = (): JSX.Element => {
+  const [content, setContent] = useState<string>('');
+  const participants = ['66bd072b001f6b1f6ac0'];
+  const [subject, setSubject] = useState<string>('');
+
+  /**
+   * To handle the form submission.
+   * @param event - Takes the inputted information and sends it via Email.
+   */
+  const handleSubmit = async (event: React.FormEvent): Promise<void> => {
+    event.preventDefault();
+    await sendEmailNotifications({ content, participants, subject });
+  };
+
   return (
     <section data-testid="admin-notifications-content">
-      <p>{`This is where I'd put my notifation dashboard, IF I HAD ONE!`}</p>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="title">Subject:</label>
+        <input
+          type="text"
+          placeholder="Enter title for email.."
+          id="title"
+          onChange={(e) => setSubject(e.target.value)}
+          data-testid="subject-text"
+        />
+        <label htmlFor="message">Content:</label>
```

Comment:
I used content because the createEmail function's prop is 'content'. So I will change everything to message (even the prop). If you see the prop changed to 'message', this is why